fixit/cli/args.py

```diff
@@ -112,7 +112,9 @@ def get_rules_parser() -> argparse.ArgumentParser:
 def relative_to_repo_root(_path: str) -> Path:
     repo_root = get_lint_config().repo_root
     try:
-        return Path(_path).resolve(strict=True).relative_to(repo_root)
```

Comment:
You don't actually want to return a relative path here.

See the default here: https://github.com/Instagram/Fixit/pull/204/files#diff-1f90d45be47f4f2f218dff3e25fe7dc12f3cda5de358a8bfecafd92a9c89affbL130 

It is an absolute path.

These args are passed to fixit.cli.find_files, which is operating relative to CWD (not repo_root!)

https://github.com/Instagram/Fixit/blob/main/fixit/cli/__init__.py#L69-L72

Reply:
This method (relative_to_repo_root) was added in #84 

The intent is good: don't allow people to specify files outside of repo_root. That is why I still call relative_to below

Reply:
Maybe we should change the name of the function. It doesn't return relative paths, it just validates that the path is under the repo root.